cmd/antictl/main.go

```diff
@@ -125,7 +126,99 @@ func main() {
 
 					},
 				},
-				// TODO (mierdin): Add command to make a session persistent
+				{
+					Name:  "persist",
+					Usage: "Make a LiveSession persistent",
+					Action: func(c *cli.Context) {
+						conn, err := grpc.Dial(fmt.Sprintf("%s:%s", host, port), grpc.WithInsecure())
+						if err != nil {
+							fmt.Println(err)
+							os.Exit(1)
+						}
+						defer conn.Close()
+						client := pb.NewLiveSessionsServiceClient(conn)
+
+						if len(c.Args()) < 2 {
+							fmt.Println("Missing args to command : antictl livesession persist <true/false> <session id>")
+							os.Exit(1)
+						}
+
+						persistent, err := strconv.ParseBool(c.Args()[0])
+						if err != nil {
+							fmt.Println(err)
+							os.Exit(1)
+						}
+						sessionid := c.Args()[1]
+
+						ls, err := client.GetLiveSession(context.Background(), &pb.LiveSession{ID: sessionid})
+						if err != nil {
+							fmt.Println(err)
+							os.Exit(1)
+						}
+
+						if ls.Persistent == persistent {
+							fmt.Printf("Persistent state is already %v, returning", persistent)
```

Comment:
```suggestion
							fmt.Printf("Persistent state is already %v, returning\n", persistent)
```

Reply:
Agreed, changed the order of args and submitted

Reply:
added the newline and submitted change